torch/nn/functional.py

```diff
@@ -2933,7 +2933,7 @@ def cross_entropy(
         >>>
         >>> # Example of target with class probabilities
         >>> input = torch.randn(3, 5, requires_grad=True)
-        >>> target = torch.randn(3, 5).softmax(dim=1)
+        >>> target = torch.randn(3, 5).softmax(dim=1).argmax(dim=1)
```

Comment:
Same as for the one below, this is expected as this example shows the soft label version.